content/source/docs/enterprise/admin/general.html.md

```diff
@@ -13,6 +13,16 @@ General settings control global behavior in Terraform Enterprise. To access gene
 
 -> **API:** See the [Admin Settings API](/docs/cloud/api/admin/settings.html).
 
+## Require Site Admins to Enable Two-factor Authentication
+
+-> **Note**: This setting can be used in conjunction with SAML in Terraform Enterprise versions v202201-1 or later. Prior to this release, the setting had no effect if SAML was also enabled
+
+Enabling this setting adds the requirement that admins have two-factor authentication enabled to access site admin functionality, which can make the site more secure.
```

Comment:
Just trying to make it easier to read :) I also wonder if we could start out with something other than using 'enabled" again. I removed that because it felt obvious, , but let me know if I've materially changed the meaning or you feel strongly we should open with that.

```suggestion
This setting can make the site more secure by requiring that admins enable two-factor authentication to access site admin functionality.
```

Reply:
Looks good to me (and easier to read)!